dbt/include/bigquery/macros/adapters/adapters.sql

```diff
@@ -0,0 +1,8 @@
+{% macro bigquery__make_temp_relation(base_relation, suffix='__dbt_tmp') %}
+    {% if var('tmp_relation_dataset', none) is none %}
+        {{ return(default__make_temp_relation(base_relation, suffix)) }}
+    {% else %}
+        {{ return(base_relation.incorporate(path={"schema": var('tmp_relation_dataset'),
+                                        "identifier": base_relation.identifier ~ suffix})) }}
+    {% endif %}
+{% endmacro %}
```

Comment:
Let's remove the configurability of the destination BQ dataset for intermediate objects to-be-dropped from this PR.

Two reasons:
1. I'd prefer not to add a new user-facing interface if we don't need to.
2. If we do end up making this configurable, we'll make it a config rather than using the `var()` macro.

```suggestion
```